core/src/main/java/tech/pegasys/eth2signer/core/signers/unencryptedfile/UnencryptedKeyFileSignerFactory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.core.signers.unencryptedfile;
+
+import tech.pegasys.eth2signer.core.signing.ArtifactSigner;
+import tech.pegasys.eth2signer.crypto.KeyPair;
+import tech.pegasys.eth2signer.crypto.SecretKey;
+
+import org.apache.tuweni.bytes.Bytes;
+
+public class UnencryptedKeyFileSignerFactory {
+
+  public static ArtifactSigner createSigner(final Bytes keyFilePath) {
```

Comment:
Think this has the wrong name, looks like the keyFileBytes? not the file path?

Reply:
done